crates/pallet-subspace/src/lib.rs

```diff
@@ -1579,6 +1582,19 @@ impl<T: Config> OnTimestampSet<T::Moment> for Pallet<T> {
     }
 }
 
+#[cfg(feature = "pot")]
+impl<T: Config> OnTimestampSet<T::Moment> for Pallet<T> {
+    fn on_timestamp_set(_moment: T::Moment) {
+        let slot_duration = Self::slot_duration();
+        assert!(
+            !slot_duration.is_zero(),
+            "Subspace slot duration cannot be zero."
+        );
+
+        // TODO: more checks
+    }
+}
```

Comment:
Why is this necessary at all? I do not think we need to check anything about timestamp in consensus anymore.